Cooper/ClientApp/src.v2/app/services/user.service.ts

```diff
@@ -15,5 +15,16 @@ export class UserService {
     return response;
   }
 
-
+  postData(user: User) {
```

Comment:
too general naming, it doesn't show what method do

Reply:
updateUser(..)?

Reply:
save/update is more obvious, yes